diff --git a/node_modules/@benjeau/react-native-draw-extras/src/Button.tsx b/node_modules/@benjeau/react-native-draw-extras/src/Button.tsx
index 5d69f3a..940b9c0 100644
--- a/node_modules/@benjeau/react-native-draw-extras/src/Button.tsx
+++ b/node_modules/@benjeau/react-native-draw-extras/src/Button.tsx
@@ -1,5 +1,5 @@
 import React from 'react';
-import { StyleProp, StyleSheet, View, ViewStyle } from 'react-native';
+import { StyleProp, StyleSheet, View, ViewStyle, TouchableOpacity } from 'react-native';
 import { RectButton } from 'react-native-gesture-handler';
 
 import { isBright } from './utils';
@@ -34,7 +34,7 @@ const Button: React.FC<ButtonProps> = ({ children, onPress, color, style }) => {
 
   return (
     <View style={containerStyle}>
-      <RectButton
+      <TouchableOpacity
         onPress={onPress}
         style={buttonStyle}
         activeOpacity={0.5}
@@ -42,7 +42,7 @@ const Button: React.FC<ButtonProps> = ({ children, onPress, color, style }) => {
         underlayColor={color}
       >
         {children}
-      </RectButton>
+      </TouchableOpacity>
     </View>
   );
 };
